### ISSUE TITLE
Future Bias

### ISSUE THREAD

**Issue by AdrianMartiR** [peerchemist/finta#25]
So I was experimenting with some data and it turned out that the VORTEX indicator does just have insane predictive performance. Except it does not. In `finta/finta.py` line 987, the it reads:
```
VMP = pd.Series(ohlc["high"] - ohlc["low"].shift(-1).abs())
```
But actually the `.shift(-1)` shifts values from 1 in the future to the present. Probably .shift(1) was intended, correct me if I'm wrong. Someone should go through the code and carefully review all ocurrences of `.shift(-n)`

**Comment by peerchemist** [comment]
Yes you are correct, it should be `.shift(1)`.

**Comment by peerchemist** [comment]
Resolved with 2f44a968d9a06eb9d647c4240872252aa8734b20
